### DISCUSSION TITLE
UNABLE TO TRIGGER WEBHOOK USING AT COMMAND

### DISCUSSION THREAD

**Discussion by Biswanath00** [esp8266/Arduino#8310] [Q&A]
I am using ESP-01 module for WIFI communication :

AT+CWJAP="FTTH-0FC5","*******"
AT+CIPSTART="TCP","maker.ifttt.com",443
AT+CIPSEND=80
>\r\nGET /trigger/help/with/key/MY_KEY HTTP/1.1\r\n
Host: maker.ifttt.com\r\n\r\n

----------------------------------------------------------------------
output:

23:57:08.737 -> +IPD,272:HTTP/1.1 400 Bad Request
23:57:08.737 -> Server: awselb/2.0
23:57:08.784 -> Date: Sun, 12 Sep 2021 18:27:08 GMT
23:57:08.831 -> Content-Type: text/html
23:57:08.831 -> Content-Length: 122
23:57:08.831 -> Connection: close
23:57:08.877 -> 
23:57:08.877 -> <html>
23:57:08.877 -> <head><title>400 Bad Request</title></head>
23:57:08.924 -> <body>
23:57:08.924 -> <center><h1>400 Bad Request</h1></center>
23:57:08.970 -> </body>
23:57:08.970 -> </html>

**Comment by d-a-v** [comment]
This is not about the arduino firmware.
Please address your question to an esp8266-AT firmware forum (example: stack overflow, esp8266.com)
